File: patch_menu2.php
<?php
$file = 'D:/dsBilling/app/Navigation/MenuRegistry.php';
$content = file_get_contents($file);
$content = preg_replace(
    '/\$prefix = request\(\)->route\(\) \? request\(\)->route\(\)->getPrefix\(\) : \'\';/',
    '$prefix = request()->route() ? ltrim(request()->route()->getPrefix() ?? "", "/") : "";',
    $content
);
file_put_contents($file, $content);
echo "Patched MenuRegistry.php with regex\n";
?>

File: patch_menu2.py
filepath = 'D:/dsBilling/app/Navigation/MenuRegistry.php'
with open(filepath, 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace("'admin.employees.index'", "'admin.employee.index'")
content = content.replace("'admin.employees.*'", "'admin.employee.*'")
content = content.replace("'admin.payrolls.index'", "'admin.payroll.index'")
content = content.replace("'admin.payrolls.*'", "'admin.payroll.*'")

with open(filepath, 'w', encoding='utf-8') as f:
    f.write(content)
